internal/tiger/cmd/integration_test.go

```diff
@@ -841,6 +841,182 @@ func TestServiceLifecycleIntegration(t *testing.T) {
+		} else {
+			t.Logf("✅ Service is correctly in PAUSED state")
+		}
+	})
+
+	t.Run("StopAlreadyStoppedService", func(t *testing.T) {
+		if serviceID == "" {
+			t.Skip("No service ID available from create test")
+		}
+
+		t.Logf("Attempting to stop already-stopped service: %s", serviceID)
+
+		output, err := executeIntegrationCommand(
+			t.Context(),
+			"service", "stop", serviceID,
+			"--wait-timeout", "10m",
+		)
+
+		// This should fail with an invalid parameters error (exit code 3)
+		if err == nil {
+			t.Errorf("Expected stop to fail for already-stopped service, but got output: %s", output)
+		} else {
+			// Check that it's an exitCodeError with ExitInvalidParameters
+			if exitErr, ok := err.(interface{ ExitCode() int }); ok {
+				if exitErr.ExitCode() != ExitInvalidParameters {
+					t.Errorf("Expected exit code %d (ExitInvalidParameters) for already-stopped service, got %d. Error: %s", ExitInvalidParameters, exitErr.ExitCode(), err.Error())
+				} else {
+					t.Logf("✅ Stop correctly failed with invalid parameters error (exit code %d) for already-stopped service", ExitInvalidParameters)
+				}
+			} else {
+				t.Errorf("Expected exitCodeError with ExitInvalidParameters exit code for already-stopped service, got: %v", err)
+			}
+		}
+	})
+
+	t.Run("StartService", func(t *testing.T) {
+		if serviceID == "" {
+			t.Skip("No service ID available from create test")
+		}
+
+		t.Logf("Starting service: %s", serviceID)
+
+		output, err := executeIntegrationCommand(
+			t.Context(),
+			"service", "start", serviceID,
+			"--wait-timeout", "10m", // Longer timeout for integration tests
+		)
+
+		if err != nil {
+			t.Fatalf("Service start failed: %v\nOutput: %s", err, output)
+		} else {
+			// Verify start success message
+			if !strings.Contains(output, "Start request accepted") &&
+				!strings.Contains(output, "ready and running") {
+				t.Errorf("Expected start success message, got: %s", output)
+			}
+			t.Logf("Service start completed successfully")
+		}
+	})
+
+	t.Run("VerifyServiceStarted", func(t *testing.T) {
+		if serviceID == "" {
+			t.Skip("No service ID available from create test")
+		}
+
+		t.Logf("Verifying service is started")
+
+		output, err := executeIntegrationCommand(t.Context(), "service", "describe", serviceID, "--output", "json")
+		if err != nil {
+			t.Fatalf("Failed to describe service after start: %v\nOutput: %s", err, output)
+		}
+
+		// Parse JSON to check status
+		var service api.Service
+		if err := json.Unmarshal([]byte(output), &service); err != nil {
+			t.Fatalf("Failed to parse service JSON: %v", err)
+		}
+
+		var status string
+		if service.Status != nil {
+			status = string(*service.Status)
+		}
+
+		t.Logf("Service status after start: %s", status)
+
+		// The status should be READY for started services
+		if status != "READY" {
+			t.Logf("Warning: Expected service status to be READY, got %s. This might be expected depending on timing or service state.", status)
```

Comment:
Again a bit of an odd error message . I think we just remove the this might be expected part no?

Reply:
Fixed here: https://github.com/timescale/tiger-cli/pull/103/commits/a71be8dadc000e210a5c6fb32aaa0fbe190a0781